Problem3.R

```diff
@@ -0,0 +1,28 @@
+# Problem 3
+
+library(ggplot2)
+
+# Histogram
+
+ggplot(galaxies, aes(x=a_26)) +
```

Comment:
Here, you assume that the script "Problem_2.R" is already loaded. If this script is not loaded, an error will occur because the "galaxies" object has not been defined. 

To deal with this, not having to write all the code from the other file again in this script, you can add this line at the top: source("Problem2.R"). This line will run the entire Problem 2 script, and in this way define the "galaxies" variable.